core/src/crds_value.rs

```diff
@@ -175,6 +244,7 @@ mod test {
             match v {
                 CrdsValueLabel::ContactInfo(_) => hits[0] = true,
                 CrdsValueLabel::Vote(_) => hits[1] = true,
+                CrdsValueLabel::EpochSlots(_) => hits[1] = true,
```

Comment:
hits[1]?

Reply:
want to grow hits to size 3 and use hits[2]?

Reply:
oops, yeah good catch